### DISCUSSION TITLE
Add more DNS nameservers to test

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [turkaysoft/glow#33] [Ideas]
Add few more popular DNS nameservers to DNS test tool. [GRC's DNS Benchmark](https://www.grc.com/dns/benchmark.htm) is a nice tool that I currently use. You can look into its [list of resolvers](http://www.grc.com/dns/resolvers.csv) for reference.

**Comment by turkaysoft** [comment]
Hello,

I can add about popular DNS services around the world that are not on Glow. If you send it with the service name, IP1 and IP2 addresses, it can be added.
